src/js/common/utils/sticky-action-box.js

```diff
@@ -0,0 +1,14 @@
+window.onload = function stickyActionBox() {
+  const actionBox = document.querySelectorAll('.show-for-large-up .sticky-action-box')[0];
+  const offset = actionBox.getBoundingClientRect();
+
+  window.addEventListener('scroll', () => {
```

Comment:
Also, I noticed a bug: if you scroll down on the page, refresh, then scroll back up to the top, the box is stuck to the top of the browser window.